app/controllers/writable_controller.rb

```diff
@@ -119,7 +120,7 @@ def display_warnings?
 
   def editor_setup
     use_javascript('posts/editor')
-    build_template_groups
+    build_template_groups(@reply.try(:user) || @post.try(:user))
```

Comment:
`@reply` must come first here because most reply controller methods set `@post` as well, and the user isn't necessarily the same.